python/taichi/aot/module.py

```diff
@@ -21,39 +62,44 @@ class Module:
     def __init__(self, arch):
         self._arch = arch
         self._kernels = []
+        self._fields = {}
         impl.get_runtime().materialize()
         self._aot_builder = impl.get_runtime().prog.make_aot_module_builder(
             arch)
 
-    def add_kernel(self, kernel_fn, name=None, template_args=None):
+    def add_field(self, name, field):
+        """Add a taichi field to the AOT module.
+      Args:
+        name: name of taichi field
+        field: taichi field
+
+        a = ti.field("something")
```

Comment:
nit: This seems as if `ti.field` takes in a string, let's use a real example

```suggestion
        a = ti.field(ti.f32, shape=(4,4))
```